k2/csrc/host_shim.cu

```diff
@@ -111,4 +110,52 @@ FsaVec FsaVecCreator::GetFsaVec() {
       arcs_);
 }
 
+Array1<bool> CheckProperties(FsaOrVec &fsas, bool (*f)(const k2host::Fsa &)) {
```

Comment:
Can we rename to`GetFsaVecBasicProperties`? To be consistent with the one in k2/csrc/fsa.h

`Check` is not a good name here as the program will not abort.

Reply:
Let's keep Check, as it just check one of those properties (GetFsaVecBasicProperties get all properties once)

Reply:
Will move the function to .cu as a local linkage